src/components/custom/_dhero.js

```diff
@@ -202,10 +202,12 @@ const query = graphql`
         }
         dtrader: file(relativePath: { eq: "dtrader_trade.png" }) {
             childImageSharp {
-                fluid(maxWidth: 1024, srcSetBreakpoints: [1440]) {
-                    ...GatsbyImageSharpFluid_withWebp_noBase64
-                    originalName
-                }
+                gatsbyImageData(
+                    quality: 70
+                    breakpoints: [360, 992, 1440]
+                    placeholder: NONE
```

Comment:
Do we want dominant colour for this placeholder?

Reply:
Updated this one and others as well 😄